ui/helpers/utils/i18n-helper.tsx

```diff
@@ -6,20 +6,25 @@ import getFetchWithTimeout from '../../../shared/modules/fetch-with-timeout';
 
 const fetchWithTimeout = getFetchWithTimeout();
 
-const warned = {};
-const missingMessageErrors = {};
-const missingSubstitutionErrors = {};
+const warned = {} as any;
+const missingMessageErrors = {} as any;
+const missingSubstitutionErrors = {} as any;
 
 /**
  * Returns a localized message for the given key
  *
- * @param {string} localeCode - The code for the current locale
- * @param {object} localeMessages - The map of messages for the current locale
- * @param {string} key - The message key
- * @param {string[]} substitutions - A list of message substitution replacements
- * @returns {null|string} The localized message
+ * @param localeCode - The code for the current locale
+ * @param localeMessages - The map of messages for the current locale
+ * @param key - The message key
+ * @param substitutions - A list of message substitution replacements
+ * @returns The localized message
  */
-export const getMessage = (localeCode, localeMessages, key, substitutions) => {
+export const getMessage = (
+  localeCode: string,
+  localeMessages: Record<string, any> | null,
```

Comment:
We actually know what the localeMessages shape will be, in this case the `I18nMessageDict` detailed above, or null.